jre.sbt

```diff
@@ -0,0 +1 @@
+ocsJreDir in ThisBuild := file("/Users/framos/jres8")
```

Comment:
Same here this is a user specific file. We may want to add this to `.gitiggnore`